DataFormats/SoATemplate/interface/SoALayout.h

```diff
@@ -745,10 +1732,12 @@
                                                                                                                        \
     /* compile-time error launched if more than one macro for methods is declared */                                   \
     static_assert(computeMethodsNumber().first <= 1,                                                                   \
-          "There can be at most one SOA_METHODS macro. Please declare all your methods inside the same macro.");       \
+          "There can be at most one SOA_ELEMENT_METHODS macro."                                                        \
+                                "Please declare all your methods inside the same macro.");                             \
```

Comment:
This is how `clang-format` would format it if it wasn't disabled:
```suggestion
                  "There can be at most one SOA_ELEMENT_METHODS macro. "                                               \
                  "Please declare all your methods inside the same macro.");                                           \
```